src/admin/__tests__/createAcls.spec.js

```diff
@@ -0,0 +1,228 @@
+              resourceName: topicName,
+              resourcePatternType: RESOURCE_PATTERN_TYPES.LITERAL,
+              principal: 'User:alice',
+              host: '*',
+              operation: OPERATION_TYPES.ALL,
+              permissionType: PERMISSION_TYPES.ALLOW,
+            },
+          ],
+        })
+      ).resolves.toEqual(true)
+    })
+
+    test('filter and fetch acl', async () => {
+      const saslConnectionOpts = () =>
+        Object.assign(sslConnectionOpts(), {
+          port: 9094,
+          sasl: {
+            mechanism: 'plain',
+            username: 'test',
+            password: 'testtest',
+          },
+        })
+      admin = createAdmin({
+        cluster: createCluster(saslConnectionOpts(), saslBrokers()),
+        logger: newLogger(),
+      })
+
+      await admin.connect()
+
+      await expect(
+        admin.describeAcls({
+          resourceName: topicName,
+          resourceType: RESOURCE_TYPES.TOPIC,
+          host: '*',
+          permissionType: PERMISSION_TYPES.ALLOW,
+          operation: OPERATION_TYPES.ANY,
+          resourcePatternTypeFilter: RESOURCE_PATTERN_TYPES.LITERAL,
+        })
+      ).resolves.toMatchObject({
+        resources: [
+          {
+            resourceType: RESOURCE_TYPES.TOPIC,
+            resourceName: topicName,
+            resourcePatternType: RESOURCE_PATTERN_TYPES.LITERAL,
+            acls: [
+              {
+                principal: 'User:alice',
+                host: '*',
+                operation: OPERATION_TYPES.ALL,
+                permissionType: PERMISSION_TYPES.ALLOW,
+              },
+            ],
+          },
+        ],
+      })
+    })
+
+    test('Try describe topic using an unauthorized user', async () => {
+      const saslConnectionOpts = () =>
+        Object.assign(sslConnectionOpts(), {
+          port: 9094,
+          sasl: {
+            mechanism: 'plain',
+            username: 'bob',
+            password: 'bobbob',
+          },
+        })
+      admin = createAdmin({
+        cluster: createCluster(saslConnectionOpts(), saslBrokers()),
+        logger: newLogger(),
+      })
+
+      await admin.connect()
+
+      await expect(admin.getTopicMetadata({ topics: [topicName] })).rejects.toThrow(
+        'Not authorized to access topics: [Topic authorization failed]'
+      )
+    })
+
+    test('Try describe topic using an authorized user', async () => {
+      const saslConnectionOpts = () =>
+        Object.assign(sslConnectionOpts(), {
+          port: 9094,
+          sasl: {
+            mechanism: 'plain',
+            username: 'alice',
+            password: 'alicealice',
+          },
+        })
+      admin = createAdmin({
+        cluster: createCluster(saslConnectionOpts(), saslBrokers()),
+        logger: newLogger(),
+      })
+
+      await admin.connect()
+
+      await expect(admin.getTopicMetadata({ topics: [topicName] })).resolves.toBeTruthy()
+    })
+
+    test('Delete ACL', async () => {
```

Comment:
This test should not depend on the other tests having run, since the order is not guaranteed and it should still pass if run with `.only`. If you need an ACL to be present before deleting it, create it in the beginning of the test.